lib/tree-view.coffee

```diff
@@ -135,6 +135,7 @@ class TreeView extends View
      'tree-view:toggle-vcs-ignored-files': -> toggleConfig 'tree-view.hideVcsIgnoredFiles'
      'tree-view:toggle-ignored-names': -> toggleConfig 'tree-view.hideIgnoredNames'
      'tree-view:remove-project-folder': (e) => @removeProjectFolder(e)
+     'tree-view:collapse-all': => @collapseAll()
```

Comment:
This would make more sense if it was placed right after `tree-view:recursive-collapse-directory`.